editor/test/integration/label_test.clj

```diff
@@ -90,10 +91,10 @@
                            calls-by-fn)]))
         render-calls-by-pass))
 
-(deftest label-batch-render
+(deftest label-batch-render-test
   (test-util/with-loaded-project
     (let [make-restore-point! #(test-util/make-graph-reverter (project/graph project))
-          add-label-component! (partial test-util/add-embedded-component! app-view (fn [node-ids] (app-view/select app-view node-ids)) (workspace/get-resource-type workspace "label"))
+          add-label-component! (partial test-util/add-embedded-component! app-view (workspace/get-resource-type workspace "label"))
```

Comment:
Got rid of the `select-fn` argument to `test-util/add-embedded-component!`, because no-one was doing anything interesting with it and I wanted `add-embedded-component!` to return the node-id of the created `EmbeddedComponentNode` regardless of what the `select-fn` was doing.